### PR TITLE
PMM-4561 Content in logs.zip

### PR DESCRIPTION
**Pull request by adivinho**

### CODE REVIEW

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-414-0432215
client docker - perconalab/pmm-client-fb:PR-414-0432215
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-414-0432215.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-414-2ff43fa
client docker - perconalab/pmm-client-fb:PR-414-2ff43fa
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-414-2ff43fa.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-414-7ee2867
client docker - perconalab/pmm-client-fb:PR-414-7ee2867
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-414-7ee2867.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-414-7f0a7dd
client docker - perconalab/pmm-client-fb:PR-414-7f0a7dd
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-414-7f0a7dd.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-414-85eb2a5
client docker - perconalab/pmm-client-fb:PR-414-85eb2a5
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-414-85eb2a5.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-414-6ddf678
client docker - perconalab/pmm-client-fb:PR-414-6ddf678
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-414-6ddf678.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-414-b11adcc
client docker - perconalab/pmm-client-fb:PR-414-b11adcc
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-414-b11adcc.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-414-06e6bf2
client docker - perconalab/pmm-client-fb:PR-414-06e6bf2
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-414-06e6bf2.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-414-112718f
client docker - perconalab/pmm-client-fb:PR-414-112718f
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-414-112718f.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-414-c472a1e
client docker - perconalab/pmm-client-fb:PR-414-c472a1e
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-414-c472a1e.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-414-5b4c593
client docker - perconalab/pmm-client-fb:PR-414-5b4c593
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-414-5b4c593.tar.gz

File: build/pmm-server-docker/playbook-init.yml [hunk 1]
```diff
@@ -36,6 +36,36 @@
         regexp: '^(\s*)listen'
         replace: '\1#listen'
 
+    - name: PMM                        | Fix nginx config
```
Comment (hors) [inline build/pmm-server-docker/playbook-init.yml]: @adivinho we have moved all playbooks to [percona-images](https://github.com/Percona-Lab/percona-images/tree/master/ansible/roles) repo. So please move all your changes connected with playbooks to it

Comment (adivinho) [reply build/pmm-server-docker/playbook-init.yml]: https://github.com/Percona-Lab/percona-images/pull/37

**Pull request closed without merging**